pallets/poa/Cargo.toml

```diff
@@ -0,0 +1,91 @@
+[package]
+name = "poa"
+version = "0.0.1"
+authors = ["Dock.io"]
+edition = "2018"
+license = 'Apache-2.0'
+[package.metadata.docs.rs]
+targets = ['x86_64-unknown-linux-gnu']
```

Comment:
Build docs for any available target (https://docs.rs/about#metadata).

Reply:
Thanks for the link. It looks like these lines are saying:

> hey https//:docs.rs, when I am published to crates.io and you build me, don't build me as all the default targets like x86_64-apple-darwin and x86_64-pc-windows-msvc. Only build me as x86_64-unknown-linux-gnu

Is that right?

Reply:
> don't build me as all the default targets 

Do you mean `targets` instead of default targets? Disregarding `targets` for now, it does exactly what I said in my comment.